tests/tests_app/test_imports.py

```diff
@@ -0,0 +1,68 @@
+import importlib
```

Comment:
This is dope ! I would move it to `lightning_utilities`.

Reply:
Yes, I think it could be useful in a few of our projects. Will upstream it in a follow-up 😃

Reply:
This is great !